lib/ui/pointer.dart

```diff
@@ -374,6 +375,9 @@ class PointerData {
   /// The current angle of the pan/zoom in radians, with 0.0 as the initial angle.
   final double rotation;
 
+  /// The identifier of the view this pointer event is for.
+  final int viewId;
```

Comment:
nit: technically, we still use Object as the type for FlutterView.viewId in this prototype, e.g. https://github.com/loic-sharma/flutter-engine/blob/bb296284a5e8364c07cce323579ede6e9566b8ef/lib/ui/window.dart#L58

Reply:
Updated!